main/commons/src/main/java/org/cryptomator/common/vaults/WebDavVolume.java

```diff
@@ -98,7 +97,12 @@ public synchronized void unmountForced() throws VolumeException {
 
 	@Override
 	public Optional<Path> getMountPoint() {
-		return Optional.ofNullable(mountPoint);
+		return Optional.empty();
```

Comment:
Isn't it possible to use a specific drive letter for WebDAV? Hence, it could use the Custom drive letter chooser

Reply:
`mountPoint` was never accessed except for this check. I figured it was legacy code.

Reply:
maybe not mount point, but drive letter. As long as it is not automatically chosen, we now the path and set the drive letter as mountpoint.

Reply:
Since WebDAVVolume does not use the mount point choosing mechanic at all, rather than "fixing" something here, this change should be reverted to keep the changes as minimal as possible.

Reply:
Reverted the discussed change for now (see 354de225edf9a91f5528b12c682c46de6f733bcf).